docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,66 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+Kubernetes allows configuring and managing resources for containers. With REANA, you can configure cluster wide **memory** limits to make the effiecient use of available resources.
```

Comment:
The setting to configure cluster-wide memory limits is just for admins and is already explained in the [Helm README](https://github.com/reanahub/reana/tree/maint-0.7/helm/reana) as `kubernetes_jobs_memory_limit`. This is not relevant for the users so I'd omit this part and explain better that a user can customise the memory limit of each workflow step in order to avoid jobs from running out of memory.